### PR TITLE
change

### PR DESCRIPTION
**Pull request by tatsuro-nishimura**
Hi,

I add an option to display only merge commits in the generated html file because we want to see only merge commits.

### CODE REVIEW

File: sphinxcontrib/vcs.py [hunk 1]
```diff
@@ -27,10 +27,15 @@ def get_revision(argument):
     return argument.strip()
 
 
+def is_merge(commit):
+    return commit.message[0:6] == 'Merged'
```
Comment (t2y) [inline sphinxcontrib/vcs.py:31]: I'm not sure how to check "Merged commit", but I think the commit message can be changed other message. Is there proper way?

**Review by t2y — CHANGES_REQUESTED** [review]

**Comment by t2y** [comment]
Why do you want to see only merge commits? How about rebasing several commits before merging them?

**Comment by tatsuro-nishimura** [comment]
Our clients can see all our commit messages in html now if we use git log as update log, but this is not preferable because they can see all our minor changes, so we want to display only merge commits.

**Comment by t2y** [comment]
> they can see all our minor changes, so we want to display only merge commits.

Does it mean that a merged commit is an important change and minor changes are not merged commit? If so, I think it's just a repository operation rule in your team/client. I wonder whether the use case is not common for most developers.

**Comment by tatsuro-nishimura** [comment]
I closed this request because we have a plan to prepare a more sophisticated pull request.

**Pull request closed without merging**